labs/bbb-webrtc-sfu/lib/mcs-core/lib/media/media-server.js

```diff
@@ -177,8 +179,7 @@ module.exports = class MediaServer extends EventEmitter {
     let mediaElement = this._mediaElements[elementId];
     let kurentoCandidate = mediaServerClient.getComplexType('IceCandidate')(candidate);
 
-    if (typeof mediaElement !== 'undefined' && typeof mediaElement.addIceCandidate === 'function' &&
-        typeof candidate !== 'undefined') {
+    if (mediaElement != null  && candidate != null) {
```

Comment:
Don't you think it's better just to do !mediaElement in all these next cases just like you did in line 28 ?

Reply:
Fixed it.